client_app/src/App.js

```diff
@@ -42,6 +44,8 @@ function App() {
           <Route path="settings" element={<Settings />} /> 
           <Route path="schedule" element={<Schedule />} />
           <Route path="request-for-help" element={<RequestForHelp />} />
+          <Route path="upcoming-requests" element={<UpcomingRequests />} />
```

Comment:
UpcomingRequests was renamed to UpcomingShifts, which is included on line 49.
```suggestion
```

I get an error when running client side app from your branch:
<img width="1475" alt="Screenshot 2025-05-05 at 9 47 04 AM" src="https://github.com/user-attachments/assets/585e37d7-8b62-443d-b276-69adcef449f5" />